tests/test_stokes.py

```diff
@@ -104,8 +109,8 @@ def NSETest(butch, stage_type, splitting):
     solver_parameters = {"mat_type": "aij",
                          "snes_type": "newtonls",
                          "snes_linesearch_type": "bt",
-                         "snes_linesearch_monitor": None,
-                         "snes_monitor": None,
+                         # "snes_linesearch_monitor": None,
+                         # "snes_monitor": None,
```

Comment:
Remove commented lines?

Reply:
These got wiped out merging `origin/master` (you made the same catch on that PR)